test/python/common.py

```diff
@@ -83,6 +83,97 @@ def assertNoLogs(self, logger=None, level=None):
         # pylint: disable=invalid-name
         return _AssertNoLogsContext(self, logger, level)
 
+    # pylint: disable=invalid-name
+    def assertDictAlmostEqual(self, dict1, dict2, delta=None, msg=None,
```

Comment:
I saw that and amended the suggestion. It is long, however.

Reply:
So I guess we could use this later on for comparing dicts of amplitudes, probabilities, etc. Since "AlmostEqual" kind of implies that we are doing numerical comparisons, and since the context is in testing, let's keep the shorter name.